cli/lsp/resolver.rs

```diff
@@ -477,14 +495,21 @@ async fn create_npm_resolver(
 fn create_node_resolver(
   npm_resolver: Option<&Arc<dyn CliNpmResolver>>,
 ) -> Option<Arc<CliNodeResolver>> {
+  use once_cell::sync::Lazy;
+
+  // it's not ideal to share this across all scopes and to
+  // never clear it, but it's fine for the time being
```

Comment:
Add a TODO and open an issue so we don't forget?

Reply:
I initially had one, but removed it because we can fix this once we get a bug that causes us to need to fix it.